cmd/exporter/exporter.go

```diff
@@ -144,31 +140,11 @@ func main() {
 	stats.InitAvailableParamAndMetrics()
 
 	if config.EnabledGPU {
-		klog.Infof("Initializing the GPU collector")
-		// the GPU operators typically takes longer time to initialize than kepler resulting in error to start the gpu driver
-		// therefore, we wait up to 1 min to allow the gpu operator initialize
-		for i := 0; i <= maxGPUInitRetry; i++ {
-			err = gpu.Init()
-			if err == nil {
-				break
-			} else {
-				time.Sleep(6 * time.Second)
+		if r := accelerator.Registry(); r != nil {
+			if err := accelerator.CreateAndRegister(accelerator.GPU, r, true); err != nil {
+				klog.Errorf("failed to init GPU accelerators: %v", err)
```

Comment:
This confuses me a bit .. here we obtain the global registry (which can't be nil) and then use  CreateAndRegister register with `r` . This feels like the Register should be a function of `registry` instead of `accelerator`